maintenance/maintenance.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
```

Comment:
Love, love this file!
I'd like to see more automation built this way instead of having to follow lengthy `oc <whatever>` instructions. 
Nice!